pom.xml

```diff
@@ -75,14 +75,14 @@
   </distributionManagement>
 
   <repositories>
-    <repository>
+    <!--repository>
```

Comment:
In other words, we need to uncomment that and make sure our nexus image provides a bintray proxy.